### ISSUE TITLE
List of keyboard shortcuts

### ISSUE THREAD

**Issue by consti** [rowanj/gitx#363]
I'm a big fan of keyboard shortcuts - is there a list of keyboard shortcuts I can use for gitx?

Thanks for the great tool - you should consider putting up a bitcoin address so people can donate some money.

**Comment by mczepiel** [comment]
Keyboard equivalents are in the menus, not sure if there are nay other shortcuts elsewhere in the app. Do you just want a reference card/PDF?

Not that I can speak for rowanj as to whether he _actually_ wants money but he does have a gittip account https://www.gittip.com/rowanj/

**Comment by consti** [comment]
@mczepiel I was asking for a reference card / PDF.
There are just a couple of shortcuts missing to make the flow perfect:
e.g. 'sign off', 'push to remote' and a way to get out of the commit message text area.

Showing them on hover (e.g. over a button: 'CMD+Enter to Commit') or next to the command (e.g. in the menu or on right click (e.g. on a unstaged file: 'CMD+S to stage')) would be perfect - I'll look into the source and maybe add them myself.

Thanks for the gittip info!

**Comment by daymun** [comment]
@consti were you able to figure out a shortcut to get out of commit message text area? I hit CMD+2 to open the stage and then I'm forced to use the mouse.

**Comment by consti** [comment]
You can press CMD+Enter to commit the staged changes.
I've hacked together a version where I can sign the commit message with a shortcut - I plan to add a couple of other shortcuts (and expose them as tooltips) - and then open a Pull-Request. :)

**Comment by daymun** [comment]
The commit shortcut is cool but is there a way to stage files without using the mouse? I know about CMD+S but not sure how to move focus from the text area.

**Comment by consti** [comment]
I think there is no way right now to focus back from the commit area
SHIFT+ALT+TAB would make sense). I'll try to implement that.

**Comment by daymun** [comment]
I tried just now and CTRL+SHIFT+TAB actually does move the focus for me! Thought I had tried everything -- either I missed it or it was recently added.

Using 0.15.1949 on Mavericks.

**Comment by consti** [comment]
You are right - thanks for finding that! It just doesn't immediately select
the first element - i.e. you'll have to press down- or up-key to select an
item.
